### PR TITLE
Added features: Support Selenium keyword "Capture Page Screenshot" upload picture to RP.

### PR DESCRIPTION
**Pull request by WandyYing**
Added features:
- attachment update
- Now, it supports the Selenium keyword "Capture Screenshot" upload picture.

fixed:
 - Encode utf8 strings. Fixes errors when non-ASCII chars are used.

change:
- get_name doc Russian to English.

### CODE REVIEW

**Comment by ailjushkin** [comment]
I have commented your initial PR

**Comment by ailjushkin** [comment]
Please review my comments.
https://github.com/ailjushkin/robotframework-reportportal-ng/pull/4

File: reportportal_listener/model.py [hunk 1]
```diff
@@ -97,11 +97,17 @@ def get_name(self):
...
-            Наименование, обрезанное до 256 символов.
+            Name, cropped up to 256 characters.
         """
-        assignment = u"{0} = ".format(", ".join(self.assign)) if self.assign else ""
-        arguments = u", ".join(self.args)
-        full_name = u"{0}{1} ({2})".format(assignment, self.name, arguments)
+        assign = ", ".join(self.assign).encode("utf8")
+        assignment = "{0} = ".format(assign) if self.assign else ""
+        arguments = ", ".join(self.args)
+        # full_name = "{0}{1} ({2})".format(assignment, self.name, arguments)
```
Comment (ailjushkin) [inline reportportal_listener/model.py]: Please remove this comment, there is no need for it.

**Review by ailjushkin — COMMENTED** [review]
Added comments

File: reportportal_listener/model.py (hunk 1 above)
Comment (WandyYing) [reply reportportal_listener/model.py]: Hello, @ailjushkin ， thanks for your review. I have been updating my code to remove this comment “# full_name = "{0}{1} ({2})".format(assignment, self.name, arguments)”.

**Comment by ailjushkin** [comment]
1.0.4.6 version uploaded on pypi

**Pull request merged**